README.md

```diff
@@ -34,6 +34,33 @@ to know more about what happens under the hood in **OpenWISP 2**.
 
 For more information see also [Applying the Unix Philosophy to Django projects: a report from the real world](https://www.slideshare.net/FedericoCapoano/applying-the-unix-philosophy-to-django-projects-a-report-from-the-real-world).
 
+System Requirements
+================
+
+The following specifications will run a new, empty instance of OpenWISP.  
+Please ensure you account for the amount of disk space your use case will require.
+
+Hardware
+---------------
+- 1GB Memory
+- 1GB disk space (excluding your database)
```

Comment:
1 GB of disk space hardly is enough for the OS alone. I would suggest 5 GB of disk space. This should be enough for many usecases and provide enough space for experimentation.
If we want to specify the minimum disk space needed, I would suggest 2 or 3 GB (I don't know exactly how much disk space an installation with this ansible playbook currently uses).

@atb00ker What do you think?

Reply:
I have a 50GB digital ocean droplet, running `df` shows roughly 48GB remaining with a running OpenWISP installation. It has 2GB RAM, running `free` shows about 1500MB free.  I agree bumping up to some recommended requirements would be reasonable.

Reply:
^ those numbers look good, thanks for sharing! 
I assumed that the 1GB is excluding the OS! Maybe we can explicitly say (1 GB free space)
Actually, the instance itself needs only a couple of 100 MBs, the main space requirements are for database and user uploaded data like floor plan images! :smile: